config/cloudformation.yaml

```
@@ -309,7 +309,7 @@ Resources:
           chown -R ${App}:mapi /var/log/${App}
           export Stack=${Stack}
           export Stage=${Stage}
-          /usr/local/node/pm2 start --uid ${App} --gid mapi /opt/${App}/server.js
+          /usr/local/node/pm2 start --uid ${App} --gid mapi --log /var/log/ /opt/${App}/server.js
```

Almost, I think it should be 
```suggestion
          /usr/local/node/pm2 start --uid ${App} --gid mapi --log /var/log/${App} /opt/${App}/server.js
```